source/iostreams.tex

```diff
@@ -2633,7 +2633,7 @@
 the function signature
 \tcode{basic_ostream\& stream::operator\shl(ios_base\& (*)(ios_base\&))}
 to permit expressions of the form
-\tcode{cout \shl dec}
+\tcode{cout \shl{} dec}
```

Comment:
I've seen them causing problems in the standard, but maybe it was just the one use in lex.tex or something like that.